src/Mayaqua/cpu_features does not contain a CMakeLists.txt file. Compile failure

Hi, there!

Thank you for using SoftEther.

Before you submit an issue, please read the following:

Is this a question?

- If the answer is "yes", then please ask your question on [www.vpnusers.com](http://www.vpnusers.com).
The issue section on GitHub is reserved for bugs and feature requests.

- If the answer is "no", please read the following:

We provide a template which is specifically made for bug reports, in order to be sure that the report includes enough details to be helpful.

Please use or adapt it as needed.

---

### Prerequisites

* [ x] Can you reproduce?
* [ x] Are you running the latest version of SoftEtherVPN?

**SoftEther version:**
Commit 93db70656d8925fe96a2790586982babb62124c1
**Component:** [Server, Client, Bridge, etc.]
Compile
**Operating system:** [Windows, Linux, BSD, macOS, etc.]
Ubuntu 16.04 from digital ocean cloud provider

**Architecture:** [64 bit, 32 bit]
64 bit

[In case it's a computer with known specs, such as the Raspberry Pi, you can specify it omitting the details.]
**Processor:** [Specify brand and model. Example: AMD Ryzen 7 1800x]

### Description

[Description of the bug]

[What you expected to happen]
The code compiles

**Actual behavior:**
Compilation error

### Steps to reproduce

1. Follow the steps on https://github.com/SoftEtherVPN/SoftEtherVPN/blob/master/src/BUILD_UNIX.md
2. Receive a configure failure and can not compile the code. 

Error:
```
Welcome to the corner-cutting configure script !

-- The C compiler identification is GNU 5.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Found Curses: /usr/lib/x86_64-linux-gnu/libcurses.so  
-- Found OpenSSL: /usr/lib/x86_64-linux-gnu/libssl.so;/usr/lib/x86_64-linux-gnu/libcrypto.so (found version "1.0.2g") 
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Found ZLIB: /usr/lib/x86_64-linux-gnu/libz.so (found version "1.2.8") 
CMake Error at src/Mayaqua/CMakeLists.txt:22 (add_subdirectory):
  The source directory

    /root/SoftEtherVPN/src/Mayaqua/cpu_features

  does not contain a CMakeLists.txt file.


-- Configuring incomplete, errors occurred!
See also "/root/SoftEtherVPN/tmp/CMakeFiles/CMakeOutput.log".
See also "/root/SoftEtherVPN/tmp/CMakeFiles/CMakeError.log".
```

This is a generated directory. Right?

--- a/src/BUILD_UNIX.md
+++ b/src/BUILD_UNIX.md
@@ -46,6 +46,7 @@ To build the programs from the source code, run the following commands:
 
 ```bash
 git clone https://github.com/SoftEtherVPN/SoftEtherVPN.git
+git submodule init && git submodule update
 cd SoftEtherVPN
 ./configure
 make -C tmp

